setup/lti-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "Canvas LTI Redirect Tool",
+  "scopes": [],
+  "extensions": [
+      {
+          "domain": "{app-hostname}",
+          "platform": "canvas.instructure.com",
```

Comment:
Need to update this value to "canvas.test.instructure.com" if connecting to Canvas test server?

Reply:
I took that config json from the LTI developer key that I created for my LTI tool on Test Canvas. so it's Canvas generated and not by me.

If you go to the LTI Key, edit, pick Paste JSON option you could see what I am talking about